bundles/sirix-rest-api/src/main/kotlin/org/sirix/rest/SirixVerticle.kt

```diff
@@ -237,7 +240,7 @@ class SirixVerticle : CoroutineVerticle() {
         put("/:database").consumes("application/xml").coroutineHandler {
             Auth(keycloak, authz, AuthRole.CREATE).handle(it)
             it.next()
-        }.handler(BodyHandler.create()).coroutineHandler {
```

Comment:
You've added the _BodyHandler_ to all POST requests. However, this one is a PUT request. I wonder if we miss something!? Not sure, though, as the tests pass, and the tests, as far as I can remember, issue a PUT when a resource is created.

Reply:
Using `BodyHandler` globally for PUT method making PUT requests caught in a deadlock situation.
![image](https://user-images.githubusercontent.com/73238797/229887350-78493dd2-70d9-4385-b633-8eb60de85292.png)

Happily, individual requests are working, so let me use `BodyHandler.create()` with each request instead of the global one. This way the design will be similar across methods and each request will have more control over itself.

Reply:
Sorry, can't follow... meaning you'll change the PR?

Reply:
Yes, we can close this one.